scripts/babel-plugin-jest-native-globals.js

```diff
@@ -14,9 +14,12 @@ module.exports = ({template}) => {
   const promiseDeclaration = template(`
     var Promise = global[Symbol.for('jest-native-promise')] || global.Promise;
   `);
+  const hrtimeDeclaration = template(`
+    var hrtime = global[Symbol.for('jest-hrtime')] || global.process.hrtime;
```

Comment:
maybe let's stick to `jest-native-*` convention. once this is replaced with Date: `jest-native-date`

Reply:
ah, good idea. will do